### PR TITLE
latex, pdflatex, pdftex, tex, texdoc, texliveonfly, tlmgr: add German translation

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages.de/common/texliveonfly.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# texliveonfly
+
+> Lade fehlende TeX Live Packages während dem Kompilieren einer `.tex` Datei herunter.
+> Mehr Informationen: <https://ctan.org/pkg/texliveonfly>.
+
+-Lade fehlende Packages während dem Kompilieren herunter:
```
Comment (bl-ue) [inline pages.de/common/texliveonfly.md:6]:
```suggestion
- Lade fehlende Packages während dem Kompilieren herunter:
```

File: pages.de/common/texdoc.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Öffne das erste Suchergebnis im Standard-PDF-Viewer:
+
+`texdoc {{suche}}`
+
+- Liste die besten Suchergebnisse auf:
+
+`texdoc --list {{suche}}`
+
+- Öffne die vollständige Dokumentation von `texdoc`
```
Comment (bl-ue) [inline pages.de/common/texdoc.md:14]:
```suggestion
- Öffne die vollständige Dokumentation von `texdoc`:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — COMMENTED** [review]
Nice work, @marchersimon! However the English versions of these pages have yet to be merged, so I'm hesitant to merge this PR since I've left some comments on that one. Not knowing German myself, I'm unsure as to whether my comments will affect the German translations or not.

**Comment by marchersimon** [comment]
@sbrl, all except for the one I left open for now only concerned English grammar. However I will proofread my translations once more and see if I can find any room for improvement.

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**